src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/AbstractMicrosoftTeamsSectionElement.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ */
+abstract class AbstractMicrosoftTeamsSectionElement implements MicrosoftTeamsSectionElementInterface
```

Comment:
I suggest removing this. 

It is only used twice and it is not providing any difficult logic.